src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -133,6 +143,13 @@ internal partial class libgpiod
         /// <returns>GPIO chip pointer handle or NULL if an error occurred.</returns>
         [DllImport(LibgpiodLibrary, SetLastError = true)]
         internal static extern SafeChipHandle gpiod_chip_open_by_number(int number);
+
+        /// <summary>
+        /// Get the API version of the library as a human-readable string.
+        /// </summary>
+        /// <returns>Human-readable string containing the library version.</returns>
+        [DllImport(LibgpiodLibrary, SetLastError = true)]
+        internal static extern IntPtr gpiod_version_string();
```

Comment:
I assume this one is available in older versions of libgpiod as well?

Reply:
I see that method in version 0.2.x 😄 :https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/tree/include/gpiod.h?h=v0.2.x#n1094

Reply:
awesome, thanks for validating @buyaa-n. In that case let's go ahead and merge this in 😄

Reply:
Thanks for the check @buyaa-n! Thanks for the review @joperezr.
I did the merge with master and resolved the conflict, forgive me for the delay.
Stay safe!

Reply:
Thank you @microhobby, merging the PR, stay safe!